frontend/lib/pages/hp-action-harassment.tsx

```diff
@@ -82,23 +81,23 @@ export const HarassmentAllegations2 = stepBuilder.createStep(props => ({
   renderIntro: renderAllegationsIntro,
   renderForm: ctx => <AllegationsFieldset>
     <CheckboxFormField {...ctx.fieldPropsFor('allegRemovedPossessions')}>
-      removed tenant possessions from the unit, or removed the unit front door or made the lock to the unit not work, or changed the lock on the unit door without giving a key to the new lock to the tenant/petitioner
+      removed your belongings, the front door, or locked you out of your apartment
     </CheckboxFormField>
     <CheckboxFormField {...ctx.fieldPropsFor('allegInducedLeaving')}>
-      offered money or valuables to tenant, or their relatives, to induce tenant to leave, or to surrender or waive their rights, without written disclosure of the tenant’s rights and without written permission to make an offer from court or the tenant; or, while: threatening, intimidating or using obscene language; frequently harassing or communicating abusively; communicating at tenant’s place of employment without prior written consent; or  knowingly falsifying or misrepresenting information to tenant
+      made buyout offers while threatening you - OR - Cursed or used profane language to intimidate you - OR - Knowingly lied to you - OR - Contacted your job without your permission
```

Comment:
Should the statements after the "- OR -"'s be un-capitalized?  (Same question goes for the checkboxes elsewhere that have "- OR -" in them)

Reply:
I'd like for the "or"s to be bold and lowercase but I didn't know how to do that. thanks for your help